index.html

```diff
@@ -66,8 +67,16 @@ <h5 id="totalDuration">00:00:00</h5>
                                 <li id="repeat_shuffle"><i class="material-icons small" title="Shuffle">shuffle</i></li>
                             </ul>
                         </div>
-                        <div id="share" class="col s3 center waves-effect btn-play-control">
-                            <i class="material-icons small" title="Copy Share Link">share</i>
+                        <div class="col s3 center dropdown-button btn-flat" data-activates="more_options">
+                            <i id="more" class="material-icons small" title="More Options">expand_more</i>
```

Comment:
Using more_vert instead of expand_more would look better I guess. I think expand_more is used to expand visible sections to full and not for extra options.